### PR TITLE
DAT-2909

### PR DESCRIPTION
**Pull request by idradm**
https://wikia-inc.atlassian.net/browse/DAT-2909

mercury part: https://github.com/Wikia/mercury/pull/1022

### CODE REVIEW

**Comment by SebastianMarzjan** [comment]
lg

**Comment by RafLeszczynski** [comment]
LGTM

File: extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php [hunk 1]
```diff
@@ -4,6 +4,7 @@ class PortableInfoboxRenderService extends WikiaService {
 	const LOGGER_LABEL = 'portable-infobox-render-not-supported-type';
 	const DESKTOP_THUMBNAIL_WIDTH = 270;
 	const MOBILE_THUMBNAIL_WIDTH = 360;
+	const MIN_BIG_IMAGE_WIDTH = 300;
```
Comment (RafLeszczynski) [inline extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php]: this const is for minimal supported image width for hero image in infobox, please consider different naming, for example: `MIN_SUPPORTED_HERO_IMG_WIDTH` or `MINIMAL_HERO_IMG_WIDTH`

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeTitle.php [hunk 2]
```diff
@@ -4,9 +4,24 @@
 class NodeTitle extends Node {
 	public function getData() {
 		if ( !isset( $this->data ) ) {
-			$this->data = [ 'value' => $this->getValueWithDefault( $this->xmlNode ) ];
+			$title = $this->sanitizeInfoboxTitle( $this->getValueWithDefault( $this->xmlNode ) );
```
Comment (RafLeszczynski) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeTitle.php]: We want to strip HTML markup from title only on mobile ( especially mercury ). This change will do it everywhere. Please limit the sanitizing to `wikiamobile` skin only

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**